lib/pure/md5.nim

```diff
@@ -162,7 +169,10 @@ proc transform(buffer: pointer, state: var MD5State) =
   state[3] = state[3] + d
 
 proc md5Init*(c: var MD5Context) =
-  ## initializes a MD5Context
+  ## Initializes a `MD5Context`.
+  ##
+  ## If you use `toMD5 proc <#toMD5,string>`_ there's no need to call this
```

Comment:
The order of procs can be changed by rearranging them, or when that is not possible because of the dependencies: forward declarations can be used to achieve custom sorting in the docs.

I'll change it to bring `toMD5` to top.

Reply:
Please remember that forward decls affect Nim's effect system. Check the docs before and after to see the inferred effects.